modules/ROOT/pages/_partials/nav-dw.adoc

```diff
@@ -258,11 +258,6 @@
 *** xref:dw-multipart-functions-form.adoc[form]
 *** xref:dw-multipart-functions-generateboundary.adoc[generateBoundary]
 *** xref:dw-multipart-types.adoc[Multipart Types]
-** xref:dw-mule.adoc[dw::Mule]
-*** xref:dw-mule-functions-causedby.adoc[causedBy]
-*** xref:dw-mule-functions-lookup.adoc[lookup]
-*** xref:dw-mule-functions-p.adoc[p]
-*** xref:dw-mule-types.adoc[Mule Types]
```

Comment:
This and the corresponding docs are missing from the data-weave repo. Is it something we manually add to the docs-dataweave?

Reply:
These functions belong to the Mule.dwl module, you need to used the [data-weave-mule-service](https://github.com/mulesoft/data-weave-mule-service) repository to generate it.
Do not remove from docs.